examples/full-app-gourmet/controller/recipe.go

```diff
@@ -35,7 +35,7 @@ func (rs recipeRessource) getAllRecipesStandardWithHelpers(w http.ResponseWriter
 		return
 	}
 
-	fuego.SendJSON(w, recipes)
+	fuego.SendJSON(w, nil, recipes)
```

Comment:
A good example would be using "r", whether it is used or not.
Users do not know the implementation.